dspace-server-webapp/src/main/resources/spring/spring-dspace-core-services.xml

```diff
@@ -108,4 +108,5 @@
         </property>
     </bean>
 
+    <bean class="org.dspace.app.rest.utils.BitstreamMetadataValuePathUtils"/>
```

Comment:
@abollini : Per @benbosman 's notes about bean loading issues, this line seems **unnecessary** to me.  If you look closely, all other `*Utils` classes in the `dspace-server-webapp` are just annotated with `@Component` instead of trying to load them via an XML spring config.  See for example:
* https://github.com/DSpace/DSpace/blob/master/dspace-server-webapp/src/main/java/org/dspace/app/rest/utils/DSpaceObjectUtils.java#L26
* https://github.com/DSpace/DSpace/blob/master/dspace-server-webapp/src/main/java/org/dspace/app/rest/utils/AuthorityUtils.java#L25

So, I suspect you should do similar here.  Remove this line from the XML config & annotate the Utils with `@Component`